src/mlpack/methods/decision_tree/gain_functions/sse_gain.hpp

```diff
@@ -25,13 +25,13 @@ namespace mlpack {
  *
  * Loss = 1 / 2 * (Observed - Predicted)^2
  */
-class SSELoss
+class SSEGain
```

Comment:
Are you sure the equation above is correct?  I read the name change from loss to gain, and wanted to check whether the function actually was gain (highest value is best) or loss (lowest value is best), but then I noticed the sentence `We will try to minimize this value while training` (which would be loss), and then the equation---but the equation doesn't seem to match the implementation.  I didn't look into the paper to remember what exactly SSEGain (or loss) is; do you think you can double-check whether the comment is accurate?  (And it may be worth double-checking the implementation too!)

Reply:
I shifted this from a previous implementation in XGBoost, where it was called `SSE_loss`. I changed it to gain because the actual implementation was just a gain function in decision trees. I'll update the comments ASAP.